code/script.js

```diff
@@ -29,23 +37,246 @@ const showMessage = (message, sender) => {
+
+  inputWrapper.innerHTML =
+  `<button id="yesBtn" type="submit" value="Yes">Yes</button>
+  <button id="noBtn" type="submit" value="No">No</button>`
+
+  let yesBtn = document.getElementById('yesBtn');
+  yesBtn.addEventListener('click', () => toConfirm(selectedTreatment, selectedDate, firstName))
+
+  let noBtn = document.getElementById('noBtn');
+  noBtn.addEventListener('click', () => denied(firstName))
+}
+
+const toConfirm = (selectedTreatment, selectedDate, firstName) => {
+  userReply('Yes');
+  botReply(`Please supply us with your email adress for final confirmation.`);
+
+  inputWrapper.innerHTML =
+  `<form id="form2">
+            <label for="userEmail">Email</label>
+            <input id="userEmail" type="email" name="userEmail" placeholder="jane@doe.com" required>
+            <button type="submit">Send</button>
+        </form>`
+
+
+  let form2 = document.getElementById('form2');
+  form2.addEventListener('submit', (event) => {
+    event.preventDefault()
+    confirmed(selectedTreatment, selectedDate, firstName)
+  })
+}
+
+
+
+const confirmed = (selectedTreatment, selectedDate, firstName) => {
+    //Transform selection strings to lowercase (for use within a sentence)
+    const selectedTreatmentValue = selectedTreatment.value;
+    const selectDateValue = selectedDate.value;
+    const lowerCaseTreatment = selectedTreatmentValue.toLowerCase();
+    const lowerCaseDate = selectDateValue.toLowerCase();
+
+  let userEmail = document.getElementById('userEmail');
+  userReply(userEmail.value);
+
+  botReply(`Thank you ${firstName}! You are welcome ${lowerCaseDate} for ${lowerCaseTreatment} with us. Have a wonderful day!`);
+
+  inputWrapper.innerHTML = `<p class="pressScissorText">Press the scissor to reload the page!</p>`;
+}
+
+const denied = (firstName) => {
+  userReply(`No`)
+  botReply(`What a shame. Maybe next time? Have a great day!`)
+
+  inputWrapper.innerHTML = `<p class="pressScissorText">Press the scissor to reload the page!</p>`;
+
+}
+
+
+const openingHours = (firstName) => {
+
+  userReply(`Opening hours`);
+  botReply(`Weekdays 10-22, 
+    Saturdays 10-18 & 
+    Sundays 11-15`)
+  botReply(`Hope to see you soon!`)
+
+      inputWrapper.innerHTML =
+      `<button id="backBtn" type="submit" value="Book appointment">Go back please</button>
+      `;
+
+      let backBtn = document.getElementById('backBtn');
+      backBtn.addEventListener('click', () => recognizeName(firstName))
+}
+
+
+const styleAdvice = (firstName) => {
+  userReply(`What should I do with my hair?`);
+
+  //console.log(`randomStyleAdvice.length`, randomStyleAdvice.length)
+  let index = getRandomNumber(0, randomStyleAdvice.length-1);
+  //console.log(`index`, index)
+
+  setTimeout(() => botReply(randomStyleAdvice[index]), 1500)
+
+  inputWrapper.innerHTML =
+      `<button id="againBtn" type="submit" value="Again">Again plz</button>
```

Comment:
I find "Again plz" and "Go back please" a bit confusing at first glance. Considering naming them something like: "I like it! Let's book it" and "Not my style" to make it more clear than one option is positive and one is negative.